test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/SStoreSuite.java

```diff
@@ -64,12 +64,12 @@ public static void main(String... args) {
     public List<HapiApiSpec> getSpecsInSuite() {
         return List.of(
                 new HapiApiSpec[] {
-                    setupAppProperties(),
-                    multipleSStoreOpsSucceed(),
+                    //                    setupAppProperties(),
+                    //                    multipleSStoreOpsSucceed(),
                     benchmarkSingleSetter(),
-                    childStorage(),
-                    temporarySStoreRefundTest(),
-                    cleanupAppProperties()
+                    //                    childStorage(),
+                    //                    temporarySStoreRefundTest(),
+                    //                    cleanupAppProperties()
```

Comment:
Are these tests commented for test purposes or are affected by the change?

Reply:
DONE